contrib/NeonVM/string.h

```diff
@@ -0,0 +1,33 @@
+#ifndef __STRING_H
+#define __STRING_H
+//#include <stdbool.h>
+#include <stdint.h>
+
+#ifdef _MSC_VER
+#ifndef strdup
+#define strdup(x) _strdup(x)
```

Comment:
Why is this #define here in the header file? It doesn't seem to be necessary for callers of these functions. If you need it in the .c file, move it to the .c file and keep it out of the header.

Reply:
Removed.